src/Api/Test/ClientMother.php

```diff
@@ -0,0 +1,98 @@
+<?php
+namespace App\Api\Test;
+
+class ClientMother
+{
+    public static function named(string $clientName): array
+    {
+        $json = [
+        'isActive' => true,
+        'maxParallelJobs' => 1,
+        'name' => $clientName,
+        'owner' => 1,
+        'quota' => -1
+        ];
+        return $json;
+    }
+    
+    public static function withAllParameters(
+        string $clientName, 
+        int $owner, 
+        string $description, 
+        bool $isActive, 
+        int $maxParallelJobs, 
+        array $postScripts, 
+        array $preScripts, 
+        int $quota, 
+        string $rsyncLongArgs, 
+        string $rsyncShortArgs, 
+        string $sshArgs, 
+        string $url
+    ): array {
+        $json = [
+            'description' => $description,
+            'isActive' => $isActive,
+            'maxParallelJobs' => $maxParallelJobs,
+            'name' => $clientName,
+            'owner' => $owner,
+            'postScripts' => $postScripts,
+            'preScripts' => $preScripts,
+            'quota' => $quota,
+            'rsyncLongArgs' => $rsyncLongArgs,
+            'rsyncShortArgs' => $rsyncShortArgs,
+            'sshArgs' => $sshArgs,
+            'url' => $url
+        ];
+        return $json;
+    }
+    
+    public static function withMaxParallelJobs(string $clientName, int $maxParallelJobs): array
```

Comment:
Create clientName internally don't ask for it. This is not "namedWithMaxParallelJobs", this is "withMaxParallelJobs".
If you really need to set an explicit name then at least make the parameter last and optional.